### PR TITLE
Docs: Capitalize bullet points in CONTRIBUTING guide

### PR DESCRIPTION
**Pull request by eklitzke**
English grammar dictates that these bullet points should be capitalized.
This also makes the capitalization style consistent with the rest of the
document, e.g. the "Decision Making Process" section.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -265,7 +265,7 @@ The project leader is the release manager for each Bitcoin Core release.
 Copyright
 ---------
 
-By contributing to this repository, you agree to license your work under the
```
Comment (promag) [inline CONTRIBUTING.md]: Unrelated change.

**Review by promag — COMMENTED** [review]
Also fix [this](https://github.com/bitcoin/bitcoin/blame/master/test/functional/README.md#L93)?

File: CONTRIBUTING.md (hunk 1 above)
Comment (eklitzke) [inline CONTRIBUTING.md]: My editor auto-trims whitespace; I'll undo this.

**Comment by eklitzke** [comment]
@promag Thanks for the feedback. I've undone the trailing whitespace cleanup at the end of the file, and also updated the bullet point you pointed out in `test/functional/README.md`.

**Comment by fanquake** [comment]
utACK 13b1e9a

**Comment by MarcoFalke** [comment]
ACK
